src/partials/subheading/do-rss.hbs

```diff
@@ -0,0 +1,9 @@
+<a class="release-notes-rss link-subheading align-center flex" href="https://docs.mulesoft.com/releases.rss" title="Add this URL to your RSS feed reader to stay up to date with MuleSoft releases.">
+    <svg width="24" height="24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 504.4 504.4" xml:space="preserve">
+        <path d="M377.6,0.2H126.4C56.8,0.2,0,57,0,126.6v251.6c0,69.2,56.8,126,126.4,126H378c69.6,0,126.4-56.8,126.4-126.4V126.6
+            C504,57,447.2,0.2,377.6,0.2z M136.8,409c-23.2,0-42-18.8-42-41.6c0-23.2,18.8-41.6,42-41.6c23.2,0,42,18.8,42,41.6
+            C178.8,390.2,160,409,136.8,409z M242,408.2c0-40-14.8-76-42.4-103.6c-28-28-63.6-42.4-103.6-42.4v-60.4
+            c112,0,206.4,94.4,206.4,206.4H242z M348.8,408.2c0-140-112.8-252.8-252.8-252.8V95c172,0,313.2,141.2,313.2,313.2H348.8z"/>
+    </svg>
+    Release Note RSS Feed
```

Comment:
should this be "Release Note**s** RSS Feed"? I know the title of that page is "Release Note Summary by Month", without the s for some reason.